app/components/views/Offset/ProjectSearch/SelectProjectButton/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC } from "react";
+import { Trans } from "@lingui/macro";
+import { Anchor, Text } from "@klimadao/lib/components";
+import { RetirementToken } from "@klimadao/lib/constants";
+import { trimWithLocale } from "@klimadao/lib/utils";
+import CheckIcon from "@mui/icons-material/Check";
+import { verra } from "@klimadao/lib/constants";
+
+import { CarbonProject } from "../../SelectiveRetirement/queryProjectDetails";
+import * as styles from "./styles";
+
+type Foobar = "balanceBCT" | "balanceNCT" | "balanceUBO" | "balanceNBO";
+
+type Props = {
+  active: boolean;
+  project: CarbonProject;
+  selectedRetirementToken: RetirementToken;
+  setSelectedProject?: (project: CarbonProject) => void;
+};
+
+export const SelectProjectButton: FC<Props> = (props) => {
+  /**
+   * Assumes following data format from respective bridges
+   *   Toucan:
+   *     projectID = "VCS-123"
+   *   C3:
+   *     projectID = "123"
+   */
+  const linkToProjectDetails = () => {
+    if (props.project.bridge === "C3") {
+      return `${verra.projectDetailPage}/${props.project.projectID}`;
+    }
+
+    if (props.project.bridge === "Toucan") {
+      const projectId = props.project.projectID.replace("VCS-", "");
+      return `${verra.projectDetailPage}/${projectId}`;
+    }
+
+    return "#";
+  };
+
+  const availableTonnes =
+    props.project[
+      `balance${props.selectedRetirementToken.toUpperCase()}` as Foobar
```

Comment:
Typescript is expecting the object property accessor to match the what's been defined in type `CarbonProject`
https://github.com/KlimaDAO/klimadao/blob/5b6f94bd6ab138888673166054907742ed06c53b/app/components/views/Offset/SelectiveRetirement/queryProjectDetails.ts#L3-L21
Need to do a bit of string manipulation here because selectedRetirementToken comes through lowercased.

Not sure how best to name it though, hence the `Foobar` placeholder haha.
Does `BalanceAttribute` kinda work here?